domains/ninjaclubreal.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "IDK",
+  "repo": "https://github.com/NINJACLUBREAL/mysite",
+  "owner": {
+    "username": "NINJACLUBREAL",
+    "email": "bee411769@gmail.com"
+  },
+"record": {
+      
+  "URL": "https://ninjaclubdev.is-a.dev"
```

Comment:
This URL doesn't exist and just redirects back to our root domain.

Reply:
change to to https://ninjaclub.rf.gd than